trollhunter-unity/Assets/FirebaseHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+// Helper class to handle Firebase-related functionalities.
+public class FirebaseHelper {
+
+	private UserAccount userAccount;
+	private List<string> textSnapshot;
+
+	// This should initialize the connection to Firebase.
+	public void Init(UserAccount userAccount, Callback callback) {

```

Comment:
I'm not sure if the callback interface is necessary here unless we make the method async (which we probably want to do).  Maybe put a TODO in there..